packages/xod-client/src/project/reducer.js

```diff
@@ -288,12 +295,16 @@ export default (state = {}, action) => {
         XP.lensPatch(patchPath), // TODO: can we have a situation where patch does not exist?
         R.compose(
           XP.assocNode(newNode),
+          // TODO: should this be in xod-project? (along with MANAGED_ATTACHMENT_TEMPLATES)
```

Comment:
The response is dictated by “Should the same behavior take place when I’m using `xodc`?” Well, I can’t add a node with `xodc` :grin: But I think it can be necessary at some point (as a tool to auto-generate something). So, I think, yes it is better suitable for `xod-project`